src/main/kotlin/edu/unq/desapp/eventeando/event/PoolMoney.kt

```diff
@@ -10,28 +10,31 @@ import java.time.LocalDate
 class PoolMoney(date: LocalDate) : Event(date) {
 
     /**
-     * TODO
+     * @return cost avg per confirmed guest
      */
     fun costPerConfirmedGuest(): Int{
         return totalCost() / confirmedGuests().size
     }
 
     /**
-     * TODO
+     * @param guest .
+     * @return balance between expenses and the cost per guest.
      */
     fun balanceOf(guest: Guest): Int{
         return totalSpendsOf(guest) - this.costPerConfirmedGuest()
     }
 
     /**
-     * TODO
+     * @param guest
+     * @return list of guest's expenses
```

Comment:
Habría que aclararbque los gastos son del guest recibido por parámetro. Pero detalles..

Reply:
totalSpend"OF"(user) me parecio que era bastante claro =P
lo hablamos cuando vengas